scripts/build_podspecs.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# Copyright 2020, gRPC Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import random
+import string
+import argparse
+
+class Dependency:
+    def __init__(self, name, version="s.version.to_s", internal=True):
```

Comment:
nit: replace "internal" with e.g. "verbatim"? Just a suggestion.

Reply:
I'm not quite following this one, could you please explain?

Reply:
Sorry, I was just wondering whether we should rename the internal parameter to something more specific, e.g. `useVerbatimVersion` to indicate that the version won't be indicated. Not feeling too strongly about it, though.

Reply:
Thats not a bad addition, the naming makes it much more straight forward why it's there. I see what you're saying! Thanks for the explanation.